index.html

```diff
@@ -39,6 +39,7 @@ <h2 class="d2l-heading-3">Components</h2>
 				<li><a href="components/button/demo/button-add.html">d2l-button-add</a></li>
 				<li><a href="components/button/demo/button-icon.html">d2l-button-icon</a></li>
 				<li><a href="components/button/demo/button-subtle.html">d2l-button-subtle</a></li>
+				<li><a href="components/button/demo/button-toggle.html">d2l-button-toggle</a></li>
```

Comment:
There's also the root README where we list out all the things (although I think we should ditch that at some point).